db/re/item_db_equip.yml

```
@@ -72624,6 +72753,35 @@ Body:
       bonus bBaseAtk,6*(min(BaseLevel,160)/10);
       if (getrefine()>=7)
          bonus bLongAtkRate,getskilllv("BA_MUSICALLESSON")+getskilllv("DC_DANCINGLESSON");
+  - Id: 18163
+    AegisName: Vigilante_Bow_J
+    Name: Vigilante Bow
+    Type: Weapon
+    SubType: Bow
+    Weight: 1000
+    Attack: 140
+    MagicAttack: 170
+    Range: 5
+    Slots: 3
+    Jobs:
+      BardDancer: false
+      Hunter: false
+      Rogue: false
```

There is still an issue here, it should be true
```suggestion
      BardDancer: true
      Hunter: true
      Rogue: true
```

---

Thanks !